src/utils/Utils.tsx

```diff
@@ -23,6 +23,13 @@ import { InactivityStatus } from '../types/Transaction';
 import User, { UserRole, UserStatus } from '../types/User';
 import Constants from './Constants';
 import Message from './Message';
+import statusMarkerAvailable from '../../assets/icon/charging_station_available.png';
```

Comment:
Use postfix Png.
statusMarkerAvailable -> statusMarkerAvailablePng
So we know that this is an image when we read the code.